controllers/class-wc-ebanx-api-controller.php

```diff
@@ -28,6 +28,25 @@ public function dashboard_check() {
 		));
 	}
 
+	/**
+	 * Responds that the plugin logs
+	 *
+	 * @return void
+	 */
+	public function retrieve_logs() {
+		header( 'Content-Type: application/json' );
+
+		if ( empty( $_GET['integration_key'] ) || $_GET['integration_key'] !== $this->get_integration_key() ) {
```

Comment:
Can we use WC_EBANX_Request::has( 'integration_key' ) and WC_EBANX_Request::read( 'integration_key' ) here so codify stops whining and to keep the same standards of the rest of the code